### ISSUE TITLE
Feature request: Automatic rest / focus timer with ambient sounds

### ISSUE THREAD

**Issue by tx-ares** [schmich/marinara#454]
Hello,  I love this app and use it daily.  I would love if the timer could automatically start rest once focus timer is finished, and vice versa start focus when rest is over.   I use the stop watch ticker sound effect during the focus times to keep me on track... during rest times it might be nice to have an ambient "peaceful" sound effect like waves on beach or light rain?   Audio cues keep me focused and using the app, thought others might like this feature too.
